apps/condo/domains/ticket/components/TicketId/TicketResidentFeatures.tsx

```diff
@@ -1,25 +1,97 @@
 import React from 'react'
-import { useIntl } from '@condo/next/intl'
 
-interface TicketResidentFeaturesProps {
-    ticket: Ticket
+interface MobileAppInstalledIndicatorProps {
+    isContactHasMobileApp: boolean
 }
 
-export const TicketResidentFeatures: React.FC<TicketResidentFeaturesProps> = ({ ticket }) => {
+const MobileAppInstalledIndicator: React.FC<MobileAppInstalledIndicatorProps> = ({ isContactHasMobileApp }) => {
     const intl = useIntl()
     const MobileAppInstalledMessage = intl.formatMessage({ id: 'pages.condo.ticket.MobileAppInstalled' })
     const MobileAppNotInstalledMessage = intl.formatMessage({ id: 'pages.condo.ticket.MobileAppNotInstalled' })
 
-    const isContactHasMobileApp = !!get(ticket, 'client')
-
     return (
         <Tooltip title={isContactHasMobileApp ? MobileAppInstalledMessage : MobileAppNotInstalledMessage}>
             <MobileIcon active={isContactHasMobileApp} />
         </Tooltip>
     )
+}
+
+interface PaymentsAvailableIndicatorProps {
+    ticketOrganizationId: string
+}
+
+const getIsPaymentsInMobileAppAvailable = (receiptsInCurrentPeriod, receiptsInPreviousPeriod) => {
+    if (receiptsInCurrentPeriod > 0 || receiptsInPreviousPeriod > 0) {
+        if (receiptsInPreviousPeriod === 0) return true
+
+        if (receiptsInCurrentPeriod >= receiptsInPreviousPeriod) {
+            return true
+        }
+    }
+
+    return false
+}
+
+const PaymentsAvailableIndicator: React.FC<PaymentsAvailableIndicatorProps> = ({ ticketOrganizationId }) => {
+    const intl = useIntl()
+    const PaymentsAvailableMessage = intl.formatMessage({ id: 'pages.condo.ticket.PaymentsAvailable' })
+    const PaymentsNotAvailableMessage = intl.formatMessage({ id: 'pages.condo.ticket.PaymentsNotAvailable' })
+
+    const currentPeriod = dayjs().startOf('month').format('YYYY-MM-DD')
+    const previousPeriod = dayjs(currentPeriod).subtract(1, 'month').format('YYYY-MM-DD')
+
+    const { count: receiptsInCurrentPeriod } = BillingReceipt.useObjects({
+        where: {
+            context: { organization: { id: ticketOrganizationId } },
+            period: currentPeriod,
+        },
+    })
+
+    const { count: receiptsInPreviousPeriod } = BillingReceipt.useObjects({
+        where: {
+            context: { organization: { id: ticketOrganizationId } },
+            period: previousPeriod,
+        },
+    })
+
+    const isPaymentsAvailable = getIsPaymentsInMobileAppAvailable(receiptsInCurrentPeriod, receiptsInPreviousPeriod)
+
+    return (
+        <Tooltip title={isPaymentsAvailable ? PaymentsAvailableMessage : PaymentsNotAvailableMessage}>
+            <BankCardIcon active={isPaymentsAvailable} />
+        </Tooltip>
+    )
+}
+
+interface TicketResidentFeaturesProps {
+    ticket: Ticket
+}
+
+const TICKET_RESIDENT_FEATURES_ROW_GUTTER: [Gutter, Gutter] = [8, 0]
+
+export const TicketResidentFeatures: React.FC<TicketResidentFeaturesProps> = ({ ticket }) => {
+    const isContactHasMobileApp = !!get(ticket, 'client')
```

Comment:
Are you sure that field `client` determines whether a contact has mobile app?
Maybe there is a `Ticket.isResidentTicket` field for that?

Reply:
No, it's a `client` field
`isResidentTicket` it's about for whom the `ticket` was created - for a resident (a contact is created), or for an employee (in this case, a contact is not created). This flag can be set, but the resident may not have a mobile application.